src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -119,7 +119,7 @@ public function getActiveModules(): array
      */
     public function getActiveModulesPaths(): array
```

Comment:
I'd go on adding a optional parameter here instead of adding a new function

Reply:
That can do the job too. Something like `useCache` with a default value to true. Will give it a try. Thanks